admin-manual/installation-setup/installation/install-advanced.rst

```diff
@@ -0,0 +1,67 @@
+.. _install-advanced:
+
+=============================
+Advanced installation options
+=============================
+
+*On this page*
+
+* :ref:`Installing for development <development>`
+* :ref:`Installing across multiple machines <multiple-machines>`
+* :ref:`Configuring Archivematica with SSL <SSL-support>`
+
+.. _development:
+
+Install for development
+-----------------------
+
+The recommended way to install Archivematica for development is with Docker.
+For instructions on how to deploy Archivematica in Docker containers, see
+the :ref:`Archivematica Docker repo <docker>`.
```

Comment:
Should this use the https://github.com/artefactual-labs/am/tree/master/compose link instead?